### PR TITLE
Use paragonie/constant_time_encoding

### PR DESCRIPTION
**Pull request by paragonie-scott**
As per the comment in #986

### CODE REVIEW

**Comment by terrafrost** [comment]
Thanks! I'm about to get on a plane but I'll try to take a look and git it merged after I land or this weekend as time permits.

**Comment by paragonie-scott** [comment]
Safe travels. :)

File: phpseclib/Net/SSH2.php [hunk 1]
```diff
@@ -4096,7 +4097,7 @@ function getServerPublicHostKey()
                 throw new NoSupportedAlgorithmsException('Unsupported signature format');
         }
 
-        return $this->signature_format . ' ' . base64_encode($this->server_public_host_key);
+        return $this->signature_format . ' ' . Base64::encode($this->server_public_host_key);
```
Comment (GrahamCampbell) [inline phpseclib/Net/SSH2.php]: this is ineeded

Comment (GrahamCampbell) [inline phpseclib/Net/SSH2.php]: this is not needed

Comment (GrahamCampbell) [inline phpseclib/Net/SSH2.php]: this information is already public

**Comment by terrafrost** [comment]
This has been merged. Net/SFTP.php didn't have the use declaration, which was causing Travis CI to fail so I added that and re-based that into your commit.

**Pull request closed without merging**